README.md

```diff
@@ -81,6 +84,26 @@ Make as much of this opensource-ready as possible:
 - Tabs component
 
 
+## How to Structure Modules for A Component
```

Comment:
How about "How to Structure Modules for Reuse" to avoid having to clarify about components here?

Reply:
👍